docs/src/docs/asciidoc/index.adoc

```diff
@@ -419,12 +445,14 @@ public class WebSecurityConfig extends WebSecurityConfigurerAdapter {
     }
 }
 ----
+====
 
 [[oauth2-boot-authorization-server-password-grant-autowired-authentication-manager]]
-==== Manually Wire An AuthenticationManager
+==== Manually Wiring An `AuthenticationManager`
 
-In the most sophisticated case, where the `AuthenticationManager` needs special configuration and you have your own `AuthenticationServerConfigurer`, then you need to do both:
+In the most sophisticated case, where the `AuthenticationManager` needs special configuration and you have your own `AuthenticationServerConfigurer`, then you need to both create your own `AuthorizationServerConfigurerAdapter` and your own WebSecurityConfigurerAdapter:
```

Comment:
Would you do the monospace ticks for `WebSecurityConfigurerAdapter` here, please?

Reply:
Also yup. Thanks again. :)